### DISCUSSION TITLE
How to generate primary PK parameters for each patient

### DISCUSSION THREAD

**Discussion by suzloup** [LAPKB/Pmetrics#142] [General]
Hi! Are you able to advise how we generate the primary PK parameters for each patient in version 2? We used to use write.table(cov.X). We can generate the data using exRes$final$data$popPoints but can only generate the first 82 patients. Thank you for your help!!

**Comment by mnneely** [comment]
If you are referring to the Bayesian mean or median posterior parameters for each subject, you have some choices.

If your results from PM_load() are in a variable called run1, then either of these will work.

run1$final$postMed or run1$final$postMean
#equivalent to final.1$postMed or final.1$postMean in legacy Pmetrics

run1$cov$summary() or run1$cov$summary(icen = "mean")
#equivalent to summary(cov.1) or summary(cov.1, icen = "mean") in legacy Pmetrics


______________________________________

Michael Neely, MD, MSc, FCP | Chief, Division of Infectious Diseases

Director, Laboratory of Applied Pharmacokinetics and Bioinformatics (www.lapk.org<http://www.lapk.org/>),
The Saban Research Institute,
Children's Hospital Los Angeles

4650 Sunset. Blvd, #MS 176 | Los Angeles, CA 90027
Ph: 323.361.5047 | Fax: 323.361.1183 | ***@***.******@***.***>

www.chla.org<http://www.chla.org/>

Professor and Clinical Scholar | Department of Pediatrics,

Keck School of Medicine | University of Southern California
www.usc.edu<http://www.usc.edu/>

________________________________
From: suzloup ***@***.***>
Sent: Wednesday, February 15, 2023 5:58:40 PM
To: LAPKB/Pmetrics ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [LAPKB/Pmetrics] How to generate primary PK parameters for each patient (Discussion #142)


Hi! Are you able to advise how we generate the primary PK parameters for each patient in version 2? We used to use write.table(cov.X). We can generate the data using exRes$final$data$popPoints but can only generate the first 82 patients. Thank you for your help!!
